conf/app.routes

```diff
@@ -26,3 +26,6 @@ GET         /api/v2/repositories                                     @uk.gov.hmr
 GET         /api/v2/repositories/:repoName                           @uk.gov.hmrc.teamsandrepositories.controller.v2.TeamsAndRepositoriesController.findRepo(repoName)
 POST        /api/v2/repositories/:repoName/branch-protection/enabled @uk.gov.hmrc.teamsandrepositories.controller.v2.TeamsAndRepositoriesController.enableBranchProtection(repoName)
 GET         /api/v2/teams                                            @uk.gov.hmrc.teamsandrepositories.controller.v2.TeamsAndRepositoriesController.allTeams()
+
+# internal
```

Comment:
could this comment be misleading? All endpoints here are for other services to call - and `platops-api` defines what is exposed